client/src/views/Help.vue

```diff
@@ -0,0 +1,83 @@
+<template class="Scroll">
+
+    <div class="Scroll">
+        <!-- <v-content>
+            <SectionOne id="section-one"/>
+            <SectionTwo id="section-two"/>     
+            <SectionThree id="section-three"/>
+        </v-content> -->
+        <v-container>
+            <div id="section-one">
```

Comment:
I wonder if this makes sense to semantically change to `<section id="rules">` as compare to the generic id here.